src/Symfony/Bundle/FrameworkBundle/Test/LoggerAssertionsTrait.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Test;
+
+use Monolog\Handler\TestHandler;
+use Monolog\Logger;
+use Monolog\LogRecord;
+
+trait LoggerAssertionsTrait
+{
+    public static function assertLogExists(string $expectedLog, string $level = Logger::DEBUG): void
```

Comment:
The signature is wrong. Monolog's level constants on the Logger class are integers, not strings.

And they are deprecated in favor of the enum in 3.x.

and in 3.x, the various `hasRecord*` methods of theTestHandler expect the Level enum.